src/pages/docs/platform-hub/installation-guide.md

```diff
@@ -1,26 +1,31 @@
 ---
 layout: src/layouts/Default.astro
 pubDate: 2025-09-30
-modDate: 2025-10-01
+modDate: 2025-10-08
 title: Installing the Public Preview of Platform Hub
-subtitle: Guide for installing a pre-release version of Octopus server
+subtitle: Guide for installing a Public Preview version of Octopus server
 icon: fa-solid fa-layer-group
 navTitle: Installation Guide
 navSection: Public Preview Installation Guide
 description: Public Preview Installation Guide for Octopus server customers
 navOrder: 33
 ---
 
-## How to install a pre-release version of Octopus server
+## How to install a Public Preview version of Octopus server
 
-:::div{.hint}
-Update September 30:
-
-We're rolling out Platform Hub to our customers slowly over the next week. We expect to have the download ready for customers by the end of this week or early next week. Please keep checking this page for updates.
-:::
-
-Self-hosted customers can install a pre-release version of Octopus Server containing Process Templates and Policies from September 30. Please visit this page on September 30 for a download and installation guide.
+Self-hosted customers can install a Public Preview version of Octopus Server containing Process Templates and Policies.
 
 :::div{.warning}
-You should only install a pre-release version of Octopus server if you are comfortable with adopting a feature that isn't fully complete. Any issues or bugs you encounter will **not** be fixed immediately. Please reach out to <sales@octopus.com> for any queries on whether this approach is right for you.
+You should only install a Public Preview version of Octopus server if you are comfortable adopting a feature before it’s fully complete. Any issues or bugs you encounter will **not** be fixed immediately. Please contact <sales@octopus.com> with any queries about whether this approach is right for you.
+:::
+1. You can download the Public Preview version of Octopus server via a direct link or by pulling a Docker image.
+- If you are running Octopus on Windows, you can download this [server release](https://download.octopusdeploy.com/octopus/Octopus.2025.4.3044-x64.msi).
+- If you are running Octopus on Linux, you can run the following Docker command
+```
+docker pull octopusdeploy/octopusdeploy:2025.4.3044-PublicPreview
+```
+2. After downloading, you can upgrade your Octopus instance with our [upgrading guide](/docs/administration/upgrading).
+3. You should now see the Platform Hub icon in your instance.
+:::div{.hint}
+Users must have **PlatformHubEdit** and **PlatformHubView** permissions enabled to access Platform Hub. These permissions can only be assigned to system teams. By default, system administrators and space managers will have **PlatformHubEdit** and **PlatformHubView** permissions enabled.
```

Comment:
nit: Should be `System manager` instead of `Space manager`.

Reply:
Oh nice catch I'll fix